### PR TITLE
Task/196 coin detail screen

### PR DESCRIPTION
**Pull request by TomasPetrik**
Click on asset wallet, then choose any network wallet from expanded list.
It should open Coin Detail Screen with balance and transactions.

### CODE REVIEW

File: .github/workflows/run_tests_on_pull_request.yml [hunk 1]
```diff
@@ -5,8 +5,7 @@ on:
 
 jobs:
   run-tests:
-    uses: SimplioOfficial/simplio-app/.github/workflows/_test_application.yml@task/187_upgrade_to_api33
-
+    uses: SimplioOfficial/simplio-app/.github/workflows/_test_application.yml@master
```
Comment (jvrastil) [inline .github/workflows/run_tests_on_pull_request.yml:8]: can you please change it in `run_tests_on_push_test_branch.yml` in a similar way?

File: lib/view/routes/authenticated_router.dart [hunk 2]
```diff
@@ -280,6 +282,17 @@ class AuthenticatedRouter with PageBuilderMixin {
                       ),
                     ),
                   ),
+                  GoRoute(
+                    path: ':assetId/:networkId/coin-detail',
+                    name: coinDetail,
+                    pageBuilder: pageBuilder(
+                      builder: (state) => CoinDetailScreen(
```
Comment (jvrastil) [inline lib/view/routes/authenticated_router.dart:289]: There is already screen prepared for this -`AssetDetailScreen`. Please use that. Also please keep those buttons from bottom bar which are used to navigation to receive, send and other assets-related page...

**Review by jvrastil — CHANGES_REQUESTED** [review]

File: lib/view/routes/authenticated_router.dart (hunk 2 above)
Comment (TomasPetrik) [reply lib/view/routes/authenticated_router.dart]: Okay. Done.

Yes I will keep this buttons with screens redirecting but in task/195 as it task was splitted.

Comment (jvrastil) [reply lib/view/routes/authenticated_router.dart]: No, keep the bottom bar in master. I understand that there will be some changes in the UI but user needs to be able to navigate to sending and receiving assets.

Comment (TomasPetrik) [reply lib/view/routes/authenticated_router.dart]: Ok I merged task/195 into this 196. You can check it out again.

**Review by jvrastil — CHANGES_REQUESTED** [review]

File: lib/view/widgets/two_lines_app_bar.dart [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                            style: SioTextStyles.bodyPrimary.apply(
+                              color: Theme.of(context).colorScheme.shadow,
+                            ),
+                          ),
+                        ],
+                      ),
+                    ),
+                  ),
+                ),
+                const SizedBox(),
```
[the remark this answers is not in the document]
Comment (TomasPetrik) [reply lib/view/widgets/two_lines_app_bar.dart:75]: No needed. Removed.

File: lib/view/widgets/earning_elevated_button.dart [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+                              style: SioTextStyles.bodyDetail.apply(
+                                color: Theme.of(context).colorScheme.onPrimary,
+                              ),
+                            ),
+                          )
+                        : Column(
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            crossAxisAlignment: CrossAxisAlignment.end,
+                            children: [
+                                Text(
```
[the remark this answers is not in the document]
Comment (TomasPetrik) [reply lib/view/widgets/earning_elevated_button.dart:100]: Okay. But as I saw this ThemedText doesn't have style bodyPrimary.

File: lib/view/widgets/coin_details_menu.dart [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    return BottomTabBarContainer(
+      height: Constants.coinsBottomTabBarHeight,
+      borderRadius: RadiusSize.radius20,
+      child: ListView.separated(
+        padding: Paddings.vertical16,
+        scrollDirection: Axis.horizontal,
+        itemBuilder: (_, index) {
+          return Row(
+            children: [
+              if (index == 0) ...{
```
[the remark this answers is not in the document]
Comment (TomasPetrik) [reply lib/view/widgets/coin_details_menu.dart:57]: There is no difference. It's just my personal habit adding curly braces while using if condition.

[the remark this answers is not in the document]
Comment (TomasPetrik) [reply lib/view/widgets/coin_details_menu.dart]: But finally I removed this curly braces for reducing code.

**Review by jvrastil — APPROVED** [review]

**Pull request merged**